pandas/core/indexes/base.py

```diff
@@ -184,11 +184,16 @@ class Index(IndexOpsMixin, PandasObject):
     See Also
     ---------
     RangeIndex : Index implementing a monotonic integer range.
-    CategoricalIndex : Index of :class:`Categorical` s.
+    CategoricalIndex : Index of class:`Categorical`s.
```

Comment:
remove the `s` after the backtick, it won't render well.

Remove all the spaces before the colon in this block